scripts/workers/remote_submission_worker.py

```diff
@@ -164,11 +168,12 @@ def create_dir_as_python_package(directory):
 
 
 def return_url_per_environment(url):
-    base_url = "http://{0}:{1}".format(DJANGO_SERVER, DJANGO_SERVER_PORT)
+    base_url = "{0}://{1}:{2}".format(DJANGO_SERVER_PROTOCOL, DJANGO_SERVER, DJANGO_SERVER_PORT)
```

Comment:
So this would become `base_url=EVALAI_API_SERVER` where `EVALAI_API_SERVER=os.environ.get("EVALAI_API_SERVER", "http://localhost:8000")`

Reply:
That makes sense, I'll make the requested changes right away ^^